containers/podman-desktop
# chore: remove random divs created by Xterm terminal

### What does this PR do?
Disposes of Xterm terminals when leaving their page to remove the extra divs created by them in the DOM

### Screenshot / video of UI



### What issues does this PR fix or reference?
Closes https://github.com/containers/podman-desktop/issues/8197



### How to test this PR?
Go to any container's or pod's log page and check that after leaving that page, the div that is added to the DOM at the end is removed.
